spyder/plugins/toolbar/plugin.py

```diff
@@ -0,0 +1,189 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""
+Toolbar Plugin.
+"""
+
+# Local imports
+from spyder.api.exceptions import SpyderAPIError
+from spyder.api.plugins import SpyderPluginV2
+from spyder.api.translations import get_translation
+from spyder.plugins.toolbar.api import ApplicationToolBars
+from spyder.plugins.toolbar.container import ToolBarContainer
+
+# Localization
+_ = get_translation('spyder')
+
+
+class ToolBar(SpyderPluginV2):
+    """
+    Docstrings viewer widget.
+    """
+    NAME = 'toolbar'
+    CONF_SECTION = NAME
+    CONF_FILE = False
+    CONTAINER_CLASS = ToolBarContainer
+    CONF_FROM_OPTIONS = {
+        'last_visible_toolbars': ('main', 'last_visible_toolbars'),
+        'toolbars_visible': ('main', 'toolbars_visible'),
+    }

```

Comment:
Please don't use options from the `main` section here (as I already mentioned a couple of times). It's better to add a new `toolbar` section to our preferences instead.

I understand that would break the user selected toolbar when moving from Spyder 4 to 5, but in this case I think it's a small price to pay to have an easier to follow code.

Reply:
Fixed.